ci/olm.sh

```diff
@@ -33,4 +33,12 @@ spec:
   channel: alpha
   source: watcher-operator-index
   sourceNamespace: openstack-operators
+  config:
```

Comment:
Adding just for reference https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/design/subscription-config.md#example